tiledb/sm/query/query.cc

```diff
@@ -793,7 +816,7 @@ Status Query::process() {
       }
       // This changes the query into INITIALIZED, but it's ok as the status
       // is updated correctly below
-      throw_if_not_ok(create_strategy(true));
+      throw_if_not_ok(create_strategy());
```

Comment:
@jp-dark Any idea why skip_checks_serialization was set to true?

Reply:
I tried to dig into it, but I really don't remember. I might have copied it from `reset_strategy_with_layout` since I am similarly updating after a change to the subarray?